modules/codegen/src/main/scala/com/twilio/guardrail/terms/collections/JavaStdLibCollections.scala

```diff
@@ -69,44 +87,93 @@ trait JavaStdLibCollections extends CollectionsAbstraction[JavaLanguage] {
         )
       )
 
+    override def empty[A]: TermHolder[JavaLanguage, MethodCallExpr, Vector[A]] =
```

Comment:
Might be useful to use `List` when representing `List`? `Vector` is closer to `Array` as far as Liskov substitution goes.

Reply:
Yeah that's fine.  I was mainly going with your `liftVectorTerm` etc. naming, but can change it.